ipa-multipoint/src/test/java/org/hyperledger/besu/nativelib/ipa_multipoint/LibIpaMultipointTest.java

```diff
@@ -75,4 +75,22 @@ public void testCallLibraryPedersenHash() {
         Bytes result = Bytes.of(LibIpaMultipoint.pedersenHash(total));
         assertThat(result).isEqualTo(Bytes32.fromHexString("0xff6e8f1877fd27f91772a4cec41d99d2f835d7320e929b8d509c5fa7ce095c51"));
     }
-}
+
+    @Test
+    public void testUpdateCommitmentSparse() {
```

Comment:
https://github.com/crate-crypto/rust-verkle/blob/bb5af2f2fe9788d49d2896b9614a3125f8227818/ffi_interface/src/lib.rs#L529. maybe we need to add this test. it seems to add only test with commitment which is identity(empty) is just testing a part of the code

Reply:
@matkt thanks, added it now